content/01-getting-started/02-setup-prisma/01-add-to-existing-project.mdx

```diff
@@ -143,24 +143,24 @@ datasource postgresql {
 
 Note that the default schema created by `prisma init` uses PostgreSQL, so you first need to switch the `provider` to `mysql`:
 
-```prisma
+```prisma file=prisma/schema.prisma
 datasource mysql {
-  provider = "mysql"
+✎  provider = "mysql"
```

Comment:
emoji has meaning here?

Reply:
Yes, it's used for the "edit" line highlight (similar to `+` and `-`). Note however that we're going to change the "syntax" for this functionality soon, more info [here](https://github.com/prisma/docs/issues/654#issuecomment-665694111).

Reply:
It basically adds a blue line highlight (and should also render the pen as an icon but [currently doesn't](https://github.com/prisma/docs/issues/740))

![image](https://user-images.githubusercontent.com/4058327/89275093-182cdb80-d642-11ea-81a5-372883ff4f3a.png)